README.md

```diff
@@ -9,7 +9,8 @@ This code is currently **work in progress** and some things are thus **known to
 * 0.0.1/minimal statement value parsing options (such as language), could lead to "interesting" things
 * Anything to do with references, qualifiers
 * Statement ranks and sitelink badges may have an undetermined behavior
-* Editing always has a fixed user and summary currently, and token validation etc does not happen
+* Property lookup by name and Statement reconciliation are currently only supported for the minimal input format.
```

Comment:
Hm, I don’t think that’s a limitation? It wouldn’t make sense to support them in the full input format IMHO, that format is supposed to be identical to the Wikibase JSON.

Reply:
:sweat_smile: ok.